report_builder/models.py

```diff
@@ -340,6 +494,13 @@ def choices_dict(self):
                 choices_dict.update({choice[0]: choice[1]})
         return choices_dict
 
+    @property
+    def choices(self):
```

Comment:
I think this is a reptation of property on line 451. unless i am going crazy, they have the same name